bench/README.md

```diff
@@ -18,40 +28,62 @@ pip3 install -r bench/requirements.txt
 
 ## Preparation
 
-Each write transactions is signed and thus requires:
+* If you are running load tests on a local pool, follow these are steps (**this item need only for the local pool**):<br>
+  * Each write transactions is signed and thus requires:
 
-* an account with write permissions (e.g. Vendor account)
-* proper values for txn `sequence` which enforces txns ordering for an account
+    * an account with write permissions (e.g. Vendor account)
+    * proper values for txn `sequence` which enforces txns ordering for an account
 
-By that reason load test uses prepared load data which can be generated as follows:
+  * Initialize the pool and test accounts (**Warning** applicable to local in-docker pool only for now):
 
-* Initialize the pool and test accounts (**Warning** applicable to local in-docker pool only for now):
+      ```bash
+      make localnet_clean
 
-    ```bash
-    make localnet_clean
+      make localnet_init
 
-    # DCL_OBSERVERS=1 make localnet_init  # to initialize observers as well
-    make localnet_init
-    
-    # ./gentestaccounts.sh [<NUM-USERS>]
-    ./gentestaccounts.sh
+      make localnet_start
+      # Note: once started ledger may require some time to complete the initialization.
+      ```
 
-    make localnet_start
-    # Note: once started ledger may require some time to complete the initialization.
-    ```
-
-* Generate test transactions:
+  * Copy local accounts keys to folder `~/.dcl/keyring-test`:
+      ```bash
+      cp ./.localnet/node0/keyring-test/* ~/.dcl/keyring-test
+      ```
 
+* Copy accounts keys to folder `~/.dcl/keyring-test` (**this item does not need for the local pool**):
+    * After connecting to the network, we must copy the `Trustee` account keys to `~/.dcl/keyring-test`. <br>
+    `The number of accounts with role Trustee must not exceed 3`.
```

Comment:
Add a sub-item:

* If you run the test against a local pool, the account keys can be copied as follows:
 ```bash
       cp ./.localnet/node0/keyring-test/* ~/.dcl/keyring-test
  ```

Reply:
Done! ✅